workflow-templates/zephyr/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report on what is not working but it should
+title: BUG - <subject title related to the issue>
+labels: 'type: bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
```suggestion
<!--- A clear and concise description of what the bug is. --->
```

Maybe add comments like you have in the feature request?

Reply:
Changed.